pandas/tseries/tests/test_base.py

```diff
@@ -29,7 +29,7 @@ def setUp(self):
     def test_ops_properties(self):
         self.check_ops_properties(['year','month','day','hour','minute','second','weekofyear','week','dayofweek','dayofyear','quarter'])
         self.check_ops_properties(['date','time','microsecond','nanosecond', 'is_month_start', 'is_month_end', 'is_quarter_start',
-                                   'is_quarter_end', 'is_year_start', 'is_year_end'], lambda x: isinstance(x,DatetimeIndex))
+                                   'is_quarter_end', 'is_year_start', 'is_year_end', 'weekday_name'], lambda x: isinstance(x,DatetimeIndex))
```

Comment:
can you add an explict test for the results of this (you can do it in ``tseries/test_timeseries.py``,
e.g. create a datetimeindex and assert that ``i.weekday_name`` matches your expectation, e.g. since you are lowercasing